src/cf-forms/src/cf-forms.less

```diff
@@ -219,6 +219,340 @@ textarea {
 
 }
 
+.input-group {
```

Comment:
this was equivalent to the new o-form-input-group: https://github.com/cfpb/capital-framework/pull/384/files#diff-e37310c249b73d03f5dde096d70e3aa9

i'll remove